sdk/core/azure-core-amqp/test/ut/amqp_value_tests.cpp

```diff
@@ -200,69 +258,102 @@ TEST_F(TestValues, TestTimestamp)
   {
     std::chrono::milliseconds timeNow{std::chrono::duration_cast<std::chrono::milliseconds>(
         std::chrono::system_clock::now().time_since_epoch())};
-    AmqpValue value{AmqpValue::CreateTimestamp(timeNow)};
-    EXPECT_EQ(timeNow, value.GetTimestamp());
+    AmqpTimestamp value{timeNow};
+    EXPECT_EQ(static_cast<std::chrono::milliseconds>(value), timeNow);
```

Comment:
BTW, should it be `operator std::chrono::milliseconds`, or `operator `std::chrono::duration`?

Reply:
The AMQP "timestamp" type is defined as a value in milliseconds.